components/Navbar/Navbar.tsx

```diff
@@ -5,9 +5,9 @@ import NavbarMobile from './NavbarMobile/NavbarMobile'
 
 export default function Navbar({ params }) {
   return (
-    <>
+    <div className="z-10 flex h-full">
```

Comment:
Hey @benalleng 

I am not able to observe any UX change with this particular change. Would you please elaborate on this? Thanks!

Also, I am still able to scroll the navbar, and I can access the content after scrolling beyond the initial viewport. I don't think this is an optimal behavior.

**Reference:**

https://github.com/saving-satoshi/saving-satoshi/assets/85434418/33206b25-e7bf-4be6-a6de-70b390bb6963

Reply:
You're correct, This was part not correctly describing the problem and part not properly fixing it, my intention is to simply have the mobile menu not add scrolling where it currently does and for it to bee the full length of the screen when scrolling already exists, which i have not fixed.

Reply:
Do you think we should disable scrolling for the navbar menu? I could see how it would be odd on a large copy page if you were to open the menu you would have a large blank space

Reply:
This particular change has to do with the functionality of having the background be an image

Reply:
> Do you think we should disable scrolling for the navbar menu?

This can be a good idea in the short run. But it can be limiting if we have a long list of navbar content and a small viewport to display it.

One solution would be to have the navbar occupy the full height of the page so no chapter content is visible when the dropdown menu is open.

Another solution could be to disable scrolling for all components except the dropdown menu when it is open.

Reply:
The usual way to handle this is to put `overflow-y: hidden` on the `<body>` element when the menu is opened, and to remove it when it is closed so users are not able to scroll when they have the menu open. Scrolling can happen inside the menu element itself, this preserves the users scroll position as well if I am not mistaken.